### PR TITLE
Add example for vmss with loadbalancer probe

### PR DESCRIPTION
**Pull request by neil-yechenwei**
fixes https://github.com/terraform-providers/terraform-provider-azurerm/issues/5947

### CODE REVIEW

File: examples/vm-scale-set/virtual_machine_scale_set/loadbalancer-probe/main.tf [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  name                           = "ssh"
+  loadbalancer_id                = azurerm_lb.example.id
+  protocol                       = "Tcp"
+  frontend_port_start            = 220
+  frontend_port_end              = 229
+  backend_port                   = 22
+  frontend_ip_configuration_name = "PublicIPAddress"
+}
+
+resource "azurerm_virtual_machine_scale_set" "example" {
```
Comment (tombuildsstuff) [inline examples/vm-scale-set/virtual_machine_scale_set/loadbalancer-probe/main.tf:68]: can we switch this over to using `azurerm_linux_virtual_machine_scale_set` - the older resources have been superseded and shouldn't be used in examples going forward

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
one comment but this otherwise LGTM 👍

File: examples/vm-scale-set/virtual_machine_scale_set/loadbalancer-probe/main.tf (hunk 1 above)
Comment (neil-yechenwei) [reply examples/vm-scale-set/virtual_machine_scale_set/loadbalancer-probe/main.tf]: updated. thx

**Comment by neil-yechenwei** [comment]
@tombuildsstuff , thanks for your comment. I've updated code.

**Review by tombuildsstuff — APPROVED** [review]
LGTM 👍

**Pull request merged**